packages/@adobe/spectrum-css-temp/components/card/index.css

```diff
@@ -44,236 +145,1091 @@ governing permissions and limitations under the License.
-  align-items: center;
-  justify-content: center;
+  &:before {
+    content: ' ';
+    inline-size: 100%;
+    block-size: 100%;
+    position: absolute;
+    inset-inline-start: 0;
+    inset-block-start: 0;
+    border-radius: var(--spectrum-card-border-radius);
+    transition: background-color var(--spectrum-global-animation-duration-100);
+  }
 
-  border-bottom: var(--spectrum-card-coverphoto-border-bottom-size) solid transparent;
-  border-radius: calc(var(--spectrum-card-border-radius) - 1px) calc(var(--spectrum-card-border-radius) - 1px) 0 0;
+  &:after {
+    content: ' ';
+    inline-size: 100%;
+    block-size: 100%;
+    position: absolute;
+    inset-inline-start: 0;
+    inset-block-start: 0;
+    border-radius: var(--spectrum-card-border-radius);
+    transition: background-color var(--spectrum-global-animation-duration-100);
+  }
 
-  background-size: cover;
-  background-position: center center;
-}
+  .spectrum-Card-grid {
+    display: grid;
+    block-size: 100%;
+    grid-template-columns: var(--spectrum-card-body-padding-left) minmax(0, auto) 1fr minmax(0, auto) var(--spectrum-card-body-padding-right);
+    grid-template-rows: minmax(var(--spectrum-card-coverphoto-height), 1fr) var(--spectrum-global-dimension-size-300) auto auto auto auto var(--spectrum-card-body-padding-bottom);
+    /* absolutely position avatar relative to the preview */
+    grid-template-areas:
+      "preview preview preview preview    preview"
+      ".       .       .       .          ."
+      ".       title   detail  actionmenu ."
+      ".       content content content    ."
+      ".       divider divider divider    ."
+      ".       footer  footer  footer     ."
+      ".       .       .       .          .";
 
-.spectrum-Card-body {
-  padding-top: var(--spectrum-card-body-padding-top);
-  padding-right: var(--spectrum-card-body-padding-right);
-  padding-bottom: var(--spectrum-card-body-padding-bottom);
-  padding-left: var(--spectrum-card-body-padding-left);
+    &:before {
+      content: ' ';
+      grid-area: preview;
+      inline-size: 100%;
+      block-size: 100%;
+      border-start-start-radius: var(--spectrum-card-border-radius);
+      border-start-end-radius: var(--spectrum-card-border-radius);
+      transition: background-color var(--spectrum-global-animation-duration-100);
+    }
+  }
+  .spectrum-Card-image,
+  .spectrum-Card-illustration {
+    inline-size: 100%;
+    block-size: 100%;
+    /*min-inline-size: var(--spectrum-card-min-width); causes to obscure border */
 
-  &:last-child {
-    border-radius: 0 0 var(--spectrum-card-border-radius) var(--spectrum-card-border-radius);
+    align-self: center;
+    justify-self: center;
+
+    border-block-end-width: var(--spectrum-card-border-size);
+    border-block-end-style: solid;
+
+    clip-path: inset(0% 0% 0% 0% round var(--spectrum-card-border-radius) var(--spectrum-card-border-radius) 0 0);
+  }
+  .spectrum-Card-image {
+  }
+  .spectrum-Card-illustration {
```

Comment:
Some duplication here with above. Should we move this class out?

Reply:
Actually there seems to be quite a bit of duplication overall...

Reply:
there's a lot of duplication, which I did intentionally, as design shifted I'd had to individually add/remove properties to very specific ones
now that we're coming to be a bit more settled, maybe i can revisit and combine
however, i think the ease of tracking down where a change should be made and what it will affect is quite useful, also, this type of pattern is close to what DSS will output

Reply:
This kind of thing would be better to have a tool that can condense for us. I was thinking of looking into css blocks by linkedin